bodhi/server/buildsys.py

```diff
@@ -179,15 +193,15 @@ def listBuildRPMs(self, id, *args, **kw):
                  'epoch': None,
                  'id': 62331,
                  'name': 'TurboGears',
-                 'nvr': 'TurboGears-1.0.2.2-2.fc7',
+                 'nvr': 'TurboGears-1.0.2.2-2.fc17',
                  'payloadhash': 'f3ec9bdce453816f94283a15a47cb952',
-                 'release': '2.fc7',
+                 'release': '2.fc17',
                  'size': 1993385,
                  'version': '1.0.2.2'},
                 ]
         if id == 16059:  # for updateinfo.xml tests
-            rpms[0]['nvr'] = rpms[1]['nvr'] = 'TurboGears-1.0.2.2-3.fc7'
-            rpms[0]['release'] = rpms[1]['release'] = '3.fc7'
+            rpms[0]['nvr'] = rpms[1]['nvr'] = 'TurboGears-1.0.2.2-3.fc17'
+            rpms[0]['release'] = rpms[1]['release'] = '3.fc17'
```

Comment:
Yeah, i had to change another set of these from fc7 to fc17 to get the fake buildsystem to give me what i needed. All the other fake turbogears packages had were fc17, so to make our fake buildsys a little more "real" i changed these ones to fc17 too.